openedx/core/djangoapps/verified_track_content/partition_scheme.py

```diff
@@ -70,7 +75,14 @@ def get_group_for_user(cls, course_key, user, user_partition, **kwargs):  # pyli
         If a course is using the Verified Track Cohorting pilot feature, this method
         returns None regardless of the user's enrollment mode.
         """
+        def _log_for_educator_1511(log_statement):
+            if unicode(course_key) == u'course-v1:ASUx+ENG101x+2177A':  # temporary logging for EDUCATOR-1511
```

Comment:
Instead of converting a course_key into unicode, why not convert the hard coded course_key string into edx course_key and then compare?